DigitalLearningSolutions.Web/Views/TrackingSystem/Shared/_DelegatesSideNavMenu.cshtml

```diff
@@ -0,0 +1,30 @@
+﻿@using Microsoft.Extensions.Configuration

```

Comment:
Should this be under Delegates/Shared folder?

Reply:
As no Delegate/Shared folder currently exists, I just put it in TrackingSystem/Shared. Would probably be better off in specifically Delegates/Shared, so sure, I'll make a folder.